README.md

```diff
@@ -68,16 +55,94 @@ Thanks to the following people who have contributed with translations:
 * Italian: Federico Pezzuolo (3XistencE-STUDIO), StarFang208
 * German: kristheb
 * Hungarian: myedition8
-* Porutgese, Brazillian: atemporal_ (Atemporal), JCraftPlay 
+* Porutgese, Brazillian: atemporal_ (Atemporal), JCraftPlay
 * Ukrainian: SmithMD24
-* Norwegian: MeatBoy 
+* Norwegian: MeatBoy
 * Czech: SilverRoll (silveroll)
 * Korean: Min-Ki Jeong
 * Chinese Simplified: ATNewHope
 * Arabic: X4Lo
 
 ## Credits
+* Original Playnite Sound Plugin by [joyrider3774](https://github.com/joyrider3774)
 * Used Icon made by [Freepik](http://www.freepik.com/)
 * Original Localization file loader by [Lacro59](https://github.com/Lacro59)
 * Sound Manager by [dfirsht](https://github.com/dfirsht)
 * Downloader Manager by [cnapolit](https://github.com/cnapolit)
+
+## Theme Integration
+Extension expose custom control to manage Music playing: ```Sounds_MusicControl```. Using this control it is possible to:
+* **Pause/Resume music** for example during trailer playback
+* **Retrieve current Music filename** to show music title
+
+### Show current Music name
+
+Current Music filename (no path, no extension) is exposed via Content.CurrentMusicName property and {PluginSettings Plugin=Sounds Path=CurrentMusicName} you may use in via binding:
+
+```xml
+    <ContentControl x:Name="Sounds_MusicControl" />
+    <TextBlock
+        Text="{Binding ElementName=Sounds_MusicControl, Path=Content.CurrentMusicName}"
+    />
+```
+
+or
+
+```xml
+    <TextBlock
+        Text="{PluginSettings Plugin=Sounds, Path=CurrentMusicName}"
+    />
+```
+
+
+
+### Pause/Resume during trailer playback
+Theme may control music track via Tag property manipulation and pause Music on 'True' value and Resume on false.
+<details>
+  <summary>Here how it can be implemented in theme GameDetail.xaml</summary>
+
+```xml
+  <Style TargetType="{x:Type GameDetails}">
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="{x:Type GameDetails}">
+                    <Grid>
+                        ...
+                        <ContentControl x:Name="Sounds_MusicControl" />
+                        ...
+                    </Grid>
+                    ...
+                    <ControlTemplate.Triggers>
+                        <DataTrigger
+                            Binding="{Binding ElementName=ExtraMetadataLoader_VideoLoaderControl_NoControls_Sound, Path=Content.IsPlayerMuted, FallbackValue={StaticResource False}}"
+                            Value="{StaticResource False}">
+                            <Setter
+                                Property="Tag"
+                                Value="{Binding ElementName=ExtraMetadataLoader_VideoLoaderControl_NoControls_Sound, Path=Content.SettingsModel.Settings.IsVideoPlaying}"
```

Comment:
Judging from this line and the use of `SettingsModel`, `Content` is the plugin object itself.
However, it's not clear how the plugin is being injected. My best guess would be Playnite itself.
Lines 117 & 126 throw me off a little, since I don't see a `IsPlayerMuted` property in any the changes in this PR.
Is this readme up to date?

Reply:
Well this part is outdated a little but as soon as now Mod injectect itself handling MediaElement.MediaOpenedEvent https://github.com/ashpynov/PlayniteSound/blob/66033f8eb7b41e590526c7bd6ace5f2f4e62fb34/PlayniteSounds.cs#L167 and https://github.com/ashpynov/PlayniteSound/blob/66033f8eb7b41e590526c7bd6ace5f2f4e62fb34/MediaElementsMonitor.cs#L47

Afrer it enumerate existing MediaElements with audio/video and pool their status.